src/views/ApartmentApp/apartmentApp.scss

```diff
@@ -1,84 +1,87 @@
 @import '../../vars';
```

Comment:
This file (apartmentApp.scss) ends up getting referenced three times in your JavaScript. `apartmentApp.css` is imported from both `src/views/ApartmentApp/components/StaffMenu/index.js` and `src/views/ApartmentApp/components/StudentApplication/components/ApplicantList/index.js`. Furthermore, `apartmentApp.scss` is imported in `src/views/ApartmentApp/index.js`. Note that the last one is the `scss` file, not the `css` file. Since `src/views/ApartmentApp/index.js` is above both of the other components, you could simplify this by importing the actual `css` file in `src/views/ApartmentApp/index.js` and then removing the (now redundant) imports in the other two files.

Reply:
Actually that sounds like it is entirely a typo and unintended mistakes that simply went unnoticed, it was only supposed to be imported in `src/views/ApartmentApp/index.js`